python/cudf/cudf/tests/conftest.py

```diff
@@ -1,10 +1,41 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+
+import os
 import pathlib
 
 import pytest
 
 import rmm  # noqa: F401
 
+_CURRENT_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
```

Comment:
This mixes approaches with the fixture `datadir` just below. Maybe use pathlib?
```suggestion
_CURRENT_DIRECTORY = str(pathlib.Path(__file__).parent)
```

Reply:
Sure. I'll stick with pathlib to avoid modifying an unrelated fixture in this PR. Note that we still need to import os for the environment modifications.